### ISSUE TITLE
[prometheus-adapter] Release 3.0.3 missing in chart repository

### ISSUE THREAD

**Issue by z0rc** [prometheus-community/helm-charts#1862]
### Describe the bug a clear and concise description of what the bug is.

https://github.com/prometheus-community/helm-charts/releases/tag/prometheus-adapter-3.0.3 is missing in from chart repo.

### What's your helm version?

irrelevant

### What's your kubectl version?

irrelevant

### Which chart?

prometheus-adapter

### What's the chart version?

3.0.3

### Enter the command that you execute and failing/misfunctioning.

```
% helm search repo prometheus-adapter
NAME                                   	CHART VERSION	APP VERSION	DESCRIPTION
prometheus-community/prometheus-adapter	3.0.2        	v0.9.1     	A Helm chart for k8s prometheus adapter
```

**Comment by calvinbui** [comment]
yep this is missing from the [index.yaml](https://prometheus-community.github.io/helm-charts/index.yaml)

**Comment by monotek** [comment]
fixed it manually: https://github.com/prometheus-community/helm-charts/commit/d87defff9bdb376d8f66242676bb4f4b331dc6ec

weird thing is it was there before: https://github.com/prometheus-community/helm-charts/commit/620ac45ab5b60fcf0c70e9234af76bf34716a078

**Comment by z0rc** [comment]
@monotek if that helps, index entry was removed here https://github.com/prometheus-community/helm-charts/commit/40aad3a8087eda7dbefb675010cd74b6aef26377
